### ISSUE TITLE
Release and integration strategy

### ISSUE THREAD

**Issue by effigies** [poldracklab/smriprep#49]
I'm pretty concerned about the approach implied by https://github.com/poldracklab/fmriprep/issues/1475#issuecomment-457317081, which indicates that fixes are being applied to sMRIPrep, a package that is not yet ready for use, and not fMRIPrep.

I would propose the following roadmap:

1) Freeze changes to sMRIPrep except as needed to ensure that it works at all.
2) Start an fMRIPrep branch that depends on the sMRIPrep code. i.e., Replace `fmriprep/workflows/anatomical.py` with `from smriprep.workflows.anatomical import *`.
3) Begin pinning fMRIPrep to specific sMRIPrep versions, similar to niworkflows.

When it comes time to split sMRIPrep out into its own app, then we can stop importing subworkflows, but I think all anatomical development should be done in one place and be accessible from fMRIPrep.

**Comment by oesteban** [comment]
With poldracklab/fmriprep#1482 we are about to resolve 1 and 2.

For 3, what do you think about the idea of having sMRIPrep and fMRIPrep versions synchronized?

There's a little of overhead (when releasing, we'll need to wait for sMRIPrep to build and deploy before fMRIPrep) but I think it will pay off.

**Comment by effigies** [comment]
tl;dr I think a little syncing will be good, but not too rigid.

IDK about synchronizing for a couple reasons:

1) dMRIPrep will not be on the same time/stability frame. I think it would be confusing to set an expectation in fMRIPrep that fMRIPrep and sMRIPrep are synced when other things that might depend on sMRIPrep won't be.
2) With higher churn in fMRIPrep (I think the majority of the work in recent months has been on functional), do we really want to induce that churn and perception of churn in sMRIPrep? I think it's okay for them to indicate different levels of stability.

That said, I do think it's a good idea to jump straight to 1.3 when you're sure we're ready. That will be an indicator that we consider it to have the same maturity as fMRIPrep at the point when they split, and that it's just a reorganization.

Possibly we could aim for a minor versions of fMRIPrep depend on the same minor version of sMRIPrep, e.g. fMRIPrep 1.3.x will be API-compatible with sMRIPrep 1.3.x, and use micro versions to patch, and minor versions to indicate larger shifts. And we might *end up* synced on the 1.x.0 ones for a while.

Anyway, maybe this would be a good discussion for Thursday? Or we could chat live, too.

**Comment by oesteban** [comment]
Let's wrap up on Thursday. In general, I agree with what you said.

**Comment by oesteban** [comment]
Please reopen if necessary, but I think this has been addressed.
